### PR TITLE
[TTAHUB-2507] Setup frontend for multiple regional dashboard types, add feature flag "training_report_dashboard"

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
- Add "training_report_dashboard
- For a user that doesn't have the flag/is not an admin, nothing will change
- For a user with the flag, or as an admin, the regional dashboard now has tabs at the top. I've left the vestigial "/" path there despite it not correlating to a tab link, so going to the base page means you will see tabs without an active link
- Setup routes at "/training-reports" and "/all-reports." These are empty dashboard pages.

## How to test
- Run migration.
- Confirm that for a user without flag/admin privileges, the experience on the regional dashboard is unchanged.
- Confirm that for an admin/flagged user, that the tab navigation appears on /regional-dashboard and that the two links take you to empty dashboard pages

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2507

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Comment by thewatermethod** [comment]
| path | <title> tag | h1 |
------|------------|---|
| '/'     | Regional Dashboard - Activity Reports                  |  Region X/Regional TTA activity dashboard   |
| '/activity-reports' | Regional Dashboard - Activity Reports | Region X/Regional Dashboard - Activity Reports | 
| '/training-reports' | Regional Dashboard - Training Reports | Region X/Regional Dashboard - Training Reports | 
| '/all-reports' | Regional Dashboard - All Reports | Region X/Regional Dashboard - All reports |

File: frontend/src/components/TabsNav.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { uniqueId } from 'lodash';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import colors from '../colors';
+import './TabsNav.scss';
+
+export default function TabsNav({ backLink, links, ariaLabel }) {
```
Comment (thewatermethod) [inline frontend/src/components/TabsNav.js:10]: This is just "RecipientTabs" made generic

File: frontend/src/components/TabsNav.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@use '../colors.scss' as *;
```
Comment (thewatermethod) [inline frontend/src/components/TabsNav.scss:1]: Content of "RecipientTabs.scss," with renamed classes

File: frontend/src/hooks/useUserDefaultRegionFilters.js [hunk 3]
```diff
@@ -31,6 +31,6 @@ export default function useUserDefaultRegionFilters(user, manageRegions) {
     defaultRegion,
     hasMultipleRegions,
     allRegionsFilters,
-    defaultFilters,
+    defaultFilters: defaultFilters(),
```
Comment (thewatermethod) [inline frontend/src/hooks/useUserDefaultRegionFilters.js:34]: This was broken before

**Review by AdamAdHocTeam — APPROVED** [review]
lgtm

**Pull request merged**